apps/datahub/src/app/record/record-apis/record-apis.component.html

```diff
@@ -1,67 +1,99 @@
+        [style.--gn-ui-button-width]="'8px'"
+        [style.--gn-ui-button-height]="'8px'"
+        [style.--gn-ui-button-padding]="'12px'"
+        [type]="isFirstStep ? 'default' : 'outline'"
+        [disabled]="isFirstStep"
+        (buttonClick)="slideToPrevious()"
+      >
+        <mat-icon
+          class="material-symbols-outlined text-[14px] text-center pt-[5px]"
+        >
+          arrow_back
+        </mat-icon>
+      </gn-ui-button>
+      <gn-ui-button
+        [style.--gn-ui-button-rounded]="'100%'"
+        [style.--gn-ui-button-width]="'8px'"
+        [style.--gn-ui-button-height]="'8px'"
+        [style.--gn-ui-button-padding]="'12px'"
+        [type]="isLastStep ? 'default' : 'outline'"
+        [disabled]="isLastStep"
+        (buttonClick)="slideToNext()"
+      >
+        <mat-icon
+          class="material-symbols-outlined text-[14px] text-center pt-[5px]"
+        >
+          arrow_forward
+        </mat-icon>
+      </gn-ui-button>
+    </div>
+  </div>
   <gn-ui-carousel
+    [itemsLength]="apiLinks.length"
+    (stepsChanged)="carouselStepsHasChanged($event)"
     containerClass="gap-4 py-10"
     stepsContainerClass="right-[var(--container-outside-width)] bottom-[calc(100%-12px)] top-auto"
   >
     <gn-ui-api-card
-      *ngFor="
-        let link of facade.apiLinks$ | async;
-        let first = first;
-        let last = last
-      "
+      *ngFor="let link of apiLinks"
       [title]="link.name"
       [link]="link"
       [currentLink]="selectedApiLink"
-      class="w-80"
+      class="w-full md:w-[320px]"
       [ngClass]="{
-        'mr-[var(--container-outside-width)]': last,
-        'ml-[var(--container-outside-width)]': first,
         'card-shadow': link !== selectedApiLink || !selectedApiLink,
         'bg-neutral-100': link === selectedApiLink
       }"
       (openRecordApiForm)="openRecordApiForm($event)"
     >
     </gn-ui-api-card>
   </gn-ui-carousel>
-</div>
-<div
-  class="content overflow-hidden transition-all duration-300"
-  [ngClass]="selectedApiLink ? 'ease-in' : 'ease-out'"
-  [style.maxHeight]="maxHeight"
-  [style.opacity]="opacity"
->
-  <div class="bg-primary-opacity-10 py-8">
-    <div class="flex flex-col px-4 gap-8 container-lg lg:mx-auto">
-      <div class="flex flex-wrap justify-between sm:mb-2 ng-star-inserted">
-        <p class="text-[21px] text-title font-title" translate>
-          record.metadata.api.form.title
-        </p>
-        <button
-          type="button"
-          class="flex items-center gap-0.5 text-primary group"
-          (click)="closeRecordApiForm()"
-        >
-          <div
-            class="text-sm font-medium opacity-50 group-hover:opacity-100 uppercase tracking-wide mt-0.5"
-            translate
-          >
-            record.metadata.api.form.closeButton
-          </div>
-          <mat-icon
-            class="!w-5 !h-5 text-xl font-bold material-symbols-outlined !flex items-center"
-            >close</mat-icon
+  <div
```

Comment:
also because of the changes in the layout, this block now doesn't use the full screen width anymore:

![image](https://github.com/geonetwork/geonetwork-ui/assets/10629150/cd261bb4-bd4e-4302-988f-af436f5043fc)

I'm not sure what the reason is to be honest :/